msteams-platform/messaging-extensions/design/messaging-extension-design.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Designing your message extension
-description: Learn how to design a Teams message extension and get the Microsoft Teams UI Kit. Describes teams design guidelines reference message extensions tips best practice
+description: Learn how to design a Teams message extension and get the Microsoft Teams UI Kit. Describes about Teams design guidelines reference message extensions tips and best practice.
```

Comment:
Not directly related to the alt-text but consider checking the images in the article `messaging-extension-design.md`. Some images can use an edit.